web/backend-fastapi/app/modules/auth/controllers/auth_controller.py

```diff
@@ -0,0 +1,44 @@
+from fastapi import APIRouter, Form, Depends
+from ..services.auth_service import AuthService
+from ..views.auth_views import AuthViews
+from ..models.auth_model import TokenRequest
+
+
+class AuthController:
```

Comment:
Are these for future outside use of this API?
Otherwise traffic from the browser doesn't perform the login through this API at the moment.